src/Displaz.jl

```diff
@@ -118,6 +127,10 @@ const _color_names = @compat Dict('r' => [1.0, 0,   0],
                                   'k' => [0.0, 0,   0],
                                   'w' => [1.0, 1,   1])
 
+const _normal_names = @compat Dict( 'x' => [1.0, 0.0, 0.0],
+                                    'y' => [0.0, 1.0, 0.0],
+                                    'z' => [0.0, 0.0, 1.0])
```

Comment:
Ok, let's kill this off then.  It can be pretty handy to have a simple abbreviation for colour (hence `_color_names`), but normals much less so.